pkg/admission/webhook/generic_webhook.go

```diff
@@ -96,7 +96,7 @@ func (p *WebhookDispatcher) getAPIBindingWorkspace(attr admission.Attributes, cl
 	for _, obj := range objs {
 		apiBinding := obj.(*apisv1alpha1.APIBinding)
 		for _, br := range apiBinding.Status.BoundResources {
-			if apiBinding.Status.BoundAPIExport != nil && apiBinding.Status.BoundAPIExport.Workspace == nil {
+			if apiBinding.Status.BoundAPIExport == nil || apiBinding.Status.BoundAPIExport.Workspace == nil {
```

Comment:
I think the old logic was intentional. Now we will see errors in the logs during bind. The right fix would probably 

Why was bound resources filled, but BoundAPIExport is unset? What does that state mean? Why can it happen?